renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Do we need `"$schema": "https://docs.renovatebot.com/renovate-schema.json",?` The bot added this, but the [guide](https://wayfair.github.io/docs/managing-dependencies/#existing-projects) doesn't mention anything about it.

Reply:
Great call-out about the guide not mentioning `$schema`! Would you mind [filing a quick issue](https://github.com/wayfair/wayfair.github.io/issues/new) on [wayfair.github.io](https://github.com/wayfair/wayfair.github.io) so we can make this clearer?

The **TL;DR** is yes, we should keep the schema in, as it can be helpful for folks who use IDEs like VSCode to edit their Renovate configuration files. By pointing to a hosted schema file, the IDE will be able to syntax-check the file for errors/hints before it's ever pushed to git!